### PR TITLE
Clean up obsolete files and README.md.

### PR DESCRIPTION
**Pull request by jmervine**

### CODE REVIEW

**Comment by XhmikosR** [comment]
Maybe delete tests/stubs too.

**Comment by XhmikosR** [comment]
And squash :P

File: README.md [hunk 1]
```diff
@@ -76,27 +53,6 @@ node make status
 tail -f ./logs/server.log
```
Comment (XhmikosR) [inline README.md]: Is this still valid?

Comment (XhmikosR) [inline README.md]: If not, .gitignore has some old entries we should remove.

Comment (jmervine) [inline README.md]: It's not, but that's actually a bug. When I setup the task in `make.js` I forgot to have it log locally. It currently logs to forevers default location which is accessible via `node ./node_modules/.bin/forever logs app.js`.

**Comment by jmervine** [comment]
You can now squash while merging, which I kind of like...

screen shot 2016-07-29 at 10 52 45 am

**Comment by XhmikosR** [comment]
I know but if I merge it and squash, will it keep you as the author?

**Comment by jmervine** [comment]
I believe so, I've used this feature a bunch in other apps and if it does mess with the author, I didn't notice it.

**Comment by XhmikosR** [comment]
Let me try it then.

**Pull request merged**